### PR TITLE
Synthetic Blood Substitute

### PR DESCRIPTION
**Pull request by DreamySkrell**
image
image

changes:
  - rscadd: "Adds Synthetic Blood Substitute."

other info:
- ~~shell IPCs have white blood (other IPCs keep their black oil blood)~~ (removed/reverted on request, shells keep black oil blood, no change here with this PR)
- organics can optionally pick the SBS blood type to get white blood
- this is intended mainly for heavily augmented organics, and you have to be have at least 8 augmented organs or limbs
- white blood has no mechanical differences (positive or negative) represented in game, as it is supposed to still just be blood (but with potentially better characteristics)
- white blood is incompatible with red blood, both ways, including organ transplantations

### CODE REVIEW

**Comment by DreamySkrell** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]
You should probably put in the in-game description that it isn't any mechanically different from regular blood

**Comment by NobleRow** [comment]
I'd like a Lore discussion tag on this until the team has seen it.

**Comment by NobleRow** [comment]
The IPC team would like the white blood be removed from Shells and kept with the black oil. We're still putting together a medical rework for IPCs and this change might be in conflict with our vision, in addition there is not currently lore to back the change for white blood over standard coolant. For the future please do ask the lore team so we don't dash hopes after it's too late. Thank you for your interest in contributing to our species, however.

**Comment by Triogenix** [comment]
So, I like this idea for organics, as I think it is interesting, and have cooked up a bit of lore about it, that I will contact you about. That said, I support what Noble said above, and would like to see this removed from shells before being merged.

**Comment by DreamySkrell** [comment]
White blood changes reverted from shell IPCs, they bleed black oil as before.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]
I don't think artificial blood should mechanically behave the same as normal blood - meaning that someone with artificial blood should need artificial blood added to their body, not normal blood. Otherwise you'd get weird inconsistencies like someone having 50% of their synthetic blood evaporated, have it replaced entirely with normal blood, and they'd still bleed only white.

**Comment by DreamySkrell** [comment]
> I don't think artificial blood should mechanically behave the same as normal blood - meaning that someone with artificial blood should need artificial blood added to their body, not normal blood. Otherwise you'd get weird inconsistencies like someone having 50% of their synthetic blood evaporated, have it replaced entirely with normal blood, and they'd still bleed only white.

Addressed. Now synth blood is its own blood type, incompatible with normal red blood.

**Comment by DreamySkrell** [comment]
Ready for review again.

**Review by Geevies — APPROVED** [review]

File: code/__DEFINES/color.dm [hunk 1]
```diff
@@ -114,8 +114,9 @@
 
 // Blood colors
 #define COLOR_HUMAN_BLOOD      "#A10808"
+#define COLOR_IPC_BLOOD        "#1F181F" // for robots with black oil "blood"
+#define COLOR_SYNTH_BLOOD      "#D9F9FF" // for organics with synthetic blood substitute
```
Comment (FluffyGhoster) [inline code/__DEFINES/color.dm:118]: DMDoc, not comments

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**